modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/components/ExportSettings.js

```diff
@@ -53,10 +74,13 @@ function ExportSettings({
 	}, [selectedExternalTypeOption]);
 
 	useEffect(() => {
-		Liferay.fire(EXPORT_FILE_FORMAT_SELECTED_EVENT, {
-			selectedExportFileFormat: selectedExternalTypeOption,
-			selectedSchema: selectedinternalClassNameKeyName,
-		});
+		if (!templateRef.current) {
+			templateRef.current = false;
```

Comment:
@jaimelr10 if the value is already false... setting it to false will do nothing.

Reply:
True, we should put this after the if!